scheduler/src/cook/rest/api.clj

```diff
@@ -296,6 +296,11 @@
   {:dataset {non-empty-max-128-characters-str non-empty-max-128-characters-str}
    (s/optional-key :partitions) #{{non-empty-max-128-characters-str non-empty-max-128-characters-str}}})
 
+(def Disk
+  "Schema for disk limit specifications"
+  {(s/optional-key :size) (s/both PosDouble (s/pred #(<= 1 % 250000) 'between-1-and-250000))
```

Comment:
I think we should make the max value configurable instead of hard-coding it.

Reply:
And move the validation to `validate-job-disk` where you can report a nice error about the maximum supported disk size.

Reply:
Also, I think we should make :size be a required key. If you include a :disk section at all, you have to include a size. You can't include an empty :disk section. @dposada  thoughts?

Reply:
Yep, requiring a request sounds reasonable to me as well